src/widgets/shared-utils.tsx

```diff
@@ -0,0 +1,14 @@
+import { newWorkspaceItem } from "@openmrs/esm-api";
```

Comment:
Duplication of this https://github.com/openmrs/openmrs-esm-patient-chart-widgets/pull/47/files

Reply:
Yeah, about this, I was going for a different approach with this PR. Often when making large scale refactors such as this one, you'll find that PR sizes get unwieldy with huge numbers of files changed. I was trying out this approach Joel suggested for keeping merge requests small and manageable https://docs.google.com/document/d/1_jCuT3vRSW8WKJbT9manso_VM0JoUIb6HpFmBi_yDMs/edit. I didn't expect that I'd come to find all the PRs merged.